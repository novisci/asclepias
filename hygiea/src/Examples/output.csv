reason,concepts,facts,begin,end
"after","home","is_funny",0,4
"after","not home",Awesome,1,10
"after","home",Awesome,0,20
