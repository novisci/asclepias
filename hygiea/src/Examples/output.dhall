List { reason : Text, concepts : Text, facts : < Awesome | NotAwesome : Text >, begin : Integer, end : Integer}
