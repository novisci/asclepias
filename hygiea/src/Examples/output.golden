"[{\"time\":[[0,4],{\"getConcepts\":\"home\",\"getFacts\":{\"contents\":\"is_funny\",\"tag\":\"NotAwesome\"}}],\"reason\":\"after\"},{\"time\":[[1,10],{\"getConcepts\":\"not home\",\"getFacts\":{\"tag\":\"Awesome\"}}],\"reason\":\"after\"},{\"time\":[[0,20],{\"getConcepts\":\"home\",\"getFacts\":{\"tag\":\"Awesome\"}}],\"reason\":\"after\"}]"